matchmaking-ext/end_participation.go

```diff
@@ -4,10 +4,19 @@ import (
 	nex "github.com/PretendoNetwork/nex-go"
 	match_making_ext "github.com/PretendoNetwork/nex-protocols-go/match-making-ext"
 	common_globals "github.com/PretendoNetwork/nex-protocols-common-go/globals"
+	"github.com/PretendoNetwork/nex-protocols-go/notifications"
 )
 
 func EndParticipation(err error, client *nex.Client, callID uint32, idGathering uint32, strMessage string) {
 	server := commonMatchMakingExtProtocol.server
+	matchmakeSession := common_globals.Sessions[idGathering].GameMatchmakeSession
+	ownerPID := matchmakeSession.Gathering.OwnerPID
+	if client.PID() == matchmakeSession.Gathering.OwnerPID {
+		if matchmakeSession.Gathering.Flags & 0x10 == 0 {
```

Comment:
What is this flag `0x10`, and do we know of any more flags? Having raw flag/type values in code like this is often unclear as to what the intention actually is

Reply:
I took the flag from the docs: https://nintendo-wiki.pretendo.network/docs/nex/protocols/match-making/types#flags

There are other flags like 0x20 and 0x200, but they are unknown

Reply:
Yeah, I agree. Maybe something like `DisconnectChangeOwner`, since it happens when the original owner disconnects?